msal/src/main/java/com/microsoft/identity/client/controllers/BrokerMSALController.java

```diff
@@ -2,21 +2,23 @@
 
 import android.content.Intent;
 
+import com.microsoft.identity.client.AuthenticationResult;
+
 public class BrokerMSALController extends MSALController {
 
     @Override
-    public void AcquireToken(MSALAcquireTokenRequest request) {
-
+    public AuthenticationResult AcquireToken(MSALAcquireTokenOperationParameters request) {
```

Comment:
Style nit: method names should be lower-cased in java -- there's a few instances so I'll just call this out once here

Reply:
Fixed.  I think i got them all.